### ISSUE TITLE
Nepal: Gorkha area

### ISSUE THREAD

**Issue by smit1678** [hotosm/imagery-requests#9]
Gorkha area tagged as priority for updated imagery: 

![](https://api.monosnap.com/rpc/file/download?id=qh5O5jfK4g0SieUeL3vyYOw3LVaIsm)

ref #6

**Comment by smit1678** [comment]
Before anyone moves on processing, @cgiovando are these still priority areas?

**Comment by cgiovando** [comment]
It looks like this is good to go, starting from the south near Gorkha, going up. I'm going to start a new layer in uMap with the areas.

**Comment by d3netxer** [comment]
We are moving ahead in processing imagery for the Gorkha area. The imagery from April 2015 has clouds in the northern section, so we are also looking at imagery from 25 Nov 2013 to cover some of the northern portion.

**Comment by cgiovando** [comment]
Great! Will you be able to add directly to the main Mapbox TMS layer?

**Comment by smit1678** [comment]
@d3netxer Are you still moving ahead with processing this area?

**Comment by d3netxer** [comment]
yes, we just finished. I asked, we can't add it to the Mapbox TMS layer. Here is the TMS:

http://hiu-maps.net/hot/1.0.0/gorkha-apr2015-flipped/{zoom}/{x}/{y}.png

I will send additional details and an updated area of the img extent that is covered on the image coord listserv

**Comment by smit1678** [comment]
@d3netxer Nice :+1:, yeah didn't expect to add to Mapbox. Best kept in a separate layer anyways.

**Comment by maning** [comment]
Used in http://tasks.hotosm.org/project/1009
